### ISSUE TITLE
[Feature request] remote controllable padding

### ISSUE THREAD

**Issue by blueForestIcarus** [kovidgoyal/kitty#2546]
It would be useful to dynamically control the window padding, with the same interface used to control other kitty parameters.
Something like:
`kitty @ set-window-padding <num>`

For example I'd like to have window padding most of the time, but write an alias to have the window padding set to zero when I open vim.

Alternatively, being able to have a different window padding setting for the alternative buffer (used by vim, less, most ncurses interfaces) would be useful. Although I don't know if this can be implemented cleanly, since the number of rows and columns would change when entering or leaving the alternative buffer.

Basically the motivation is that I want window_padding when using kitty with a shell, but no padding when using it with an ncurses interface, and I want to be able to switch back and forth.
